bin/cli.js

```diff
@@ -261,8 +284,8 @@ const argv = yargs
   .middleware([
     setLogLevel,
     setRequestHeaders,
-    loadConfigMiddleware,
     injectAccountIdMiddleware,
+    loadConfigMiddleware,
```

Comment:
Need to move this above `injectAccountIdMiddleware` and add the order needs to be:

`loadConfigMiddleware`
`injectAccountIdMiddleware`
`validateAccountOptions`